src/hex_boosting.proto

```diff
@@ -17,10 +17,10 @@ message boosted_hex_info_v1 {
   // the multipliers valid for this hex
   // for each period
   repeated uint32 multipliers = 5;
-  // the onchain pubkey of the boosted hex account
-  bytes boosted_hex_pubkey = 6;
-  // the onchain pubkey of the boost config account
-  bytes boost_config_pubkey = 7;
+  // the onchain address of the boosted hex account
+  string boosted_hex_pubkey = 6;
```

Comment:
these need to be bytes..

Reply:
why so, on chain they are varchar?

Reply:
the varchar contains stringified version of the pubkey ie `BZM1QTud72B2cpTW7PhEnFmRX7ZWzvY7DpPpNJJuDrWG`

i can put them back to bytes in the proto, i have no requirements to have them in the proto as string...i was just keeping them in the original format...